napari/layers/_markers_layer/_constants.py

```diff
@@ -0,0 +1,31 @@
+from enum import Enum
+
+
+class Symbols(Enum):
```

Comment:
I think `Symbol` makes more sense. @jni what are your thoughts on plural vs singular enum family names?

Reply:
Good point. I looked around a bit and singular definitely looks more standard. I went ahead and changed the name to singular. @jni, let me know if you have a strong preference the other way.